yb-voyager/src/callhome/diagnostics.go

```diff
@@ -33,35 +32,122 @@ import (
 
 // call-home json formats
 var (
-	jsonFilePath    string
-	Payload         payload
 	SendDiagnostics utils.BoolStr
 )
 
-const (
+var (
 	CALL_HOME_SERVICE_HOST = "34.105.39.253"
 	CALL_HOME_SERVICE_PORT = 80
 )
 
-type payload struct {
-	MigrationUuid         uuid.UUID `json:"UUID"`
-	StartTime             string    `json:"start_time"`
-	YBVoyagerVersion      string    `json:"yb_voyager_version"`
-	LastUpdatedTime       string    `json:"last_updated_time"`
-	SourceDBType          string    `json:"source_db_type"`
-	SourceDBVersion       string    `json:"source_db_version"`
-	Issues                string    `json:"issues"`
-	DBObjects             string    `json:"database_objects"`
-	TargetDBVersion       string    `json:"target_db_version"`
-	NodeCount             int       `json:"node_count"`
-	ParallelJobs          int       `json:"parallel_jobs"`
-	TotalRows             int64     `json:"total_rows"`
-	TotalSize             int64     `json:"total_size"`
-	LargestTableRows      int64     `json:"largest_table_rows"`
-	LargestTableSize      int64     `json:"largest_table_size"`
-	TargetClusterLocation string    `json:"target_cluster_location"` //TODO
-	TargetDBCores         int       `json:"target_db_cores"`         //TODO
-	SourceCloudDBType     string    `json:"source_cloud_type"`       //TODO
+/*
+Call-home diagnostics table structure -
+CREATE TABLE diagnostics (
+
+	migration_uuid UUID,
+	phase_start_time TIMESTAMP WITH TIME ZONE,
+	source_db_details JSONB,
+	target_db_details JSONB,
+	yb_voyager_version TEXT,
+	migration_phase TEXT,
+	phase_payload JSONB,
+	migration_type TEXT,
+	time_taken_sec NUMERIC(20,2),
+	status TEXT,
+	PRIMARY KEY (migration_uuid, phase_start_time, migration_phase)
+
+);
+*/
+type Payload struct {
+	MigrationUUID    uuid.UUID `json:"migration_uuid"`
+	PhaseStartTime   string    `json:"phase_start_time"`
+	SourceDBDetails  string    `json:"source_db_details"`
+	TargetDBDetails  string    `json:"target_db_details"`
+	YBVoyagerVersion string    `json:"yb_voyager_version"`
+	MigrationPhase   string    `json:"migration_phase"`
+	PhasePayload     string    `json:"phase_payload"`
+	MigrationType    string    `json:"migration_type"`
+	TimeTakenSec     float64   `json:"time_taken_sec"`
+	Status           string    `json:"status"`
+}
+
+type SourceDBDetails struct {
+	Host      string `json:"host"`
+	DBType    string `json:"db_type"`
+	DBVersion string `json:"db_version"`
+	DBSize    int64  `json:"total_db_size"` //bytes
+
+}
+
+type TargetDBDetails struct {
+	Host      string `json:"host"`
+	DBVersion string `json:"db_version"`
+	NodeCount int    `json:"node_count"`
+	Cores     int    `json:"cores"`
+}
+
+type AssessMigrationPhasePayload struct {
```

Comment:
We should send `SchemaSummary` also here. Because we wont be having the analyze schema output at this point.